cmd/broker/cmd/cmd.go

```diff
@@ -0,0 +1,128 @@
+//  Copyright (c) 2017-2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/uber/aresdb/broker"
+	"github.com/uber/aresdb/broker/config"
+	"github.com/uber/aresdb/cmd/aresd/cmd"
+	"github.com/uber/aresdb/common"
+	"github.com/uber/aresdb/cluster/topology"
+	"github.com/uber/aresdb/controller/client"
+	"github.com/uber/aresdb/utils"
+	"time"
+)
+
+func Execute(setters ...cmd.Option) {
```

Comment:
better to use fx instead of migration later

Reply:
not sure I get it..  do you mean using array?

Reply:
it is fine, as long as from internal package we can use uberfx to call